### PR TITLE
Load searchable points with correct filepath when cwd != project root

### PR DESCRIPTION
**Pull request by jimbofreedman**
I'm not sure if this is something you want or if it just stays in our fork, but a lot of spiders won't run on scrapinghub because the working directory isn't the project root.

If it is, then we'll need to make a few changes to `setup.py` obviously.

### CODE REVIEW

File: locations/spiders/cookout.py [hunk 1]
```diff
@@ -16,7 +19,10 @@ class CookoutSpider(scrapy.Spider):
     def start_requests(self):
         base_url = "https://cookout.com/wp-admin/admin-ajax.php?action=store_search&lat={lat}&lng={lng}&max_results=300&search_radius=500"
 
-        with open("./locations/searchable_points/us_centroids_100mile_radius_state.csv") as points:
+        logging.info(os.path.dirname(os.path.realpath(__file__)))
+        logging.info(os.getcwd())
```
Comment (iandees) [inline locations/spiders/cookout.py:23]: Remove debug?

**Comment by iandees** [comment]
I think I'm ok with merging this. What changes to setup.py are you thinking?

**Comment by jimbofreedman** [comment]
> I think I'm ok with merging this. What changes to setup.py are you thinking?

Nothing more than a deliberately chosen `name` and `version` 😁

**Comment by iandees** [comment]
Gotcha. I don't really want to release this as a module and worry about versioning. Is the setup.py really required for scrapinghub?

**Comment by jimbofreedman** [comment]
Yes - it autogenerates on deployment, but the crucial line is the extra one that adds `include_package_data=True`. 

Perhaps we could merge the spider changes and `__init.py__` - that's enough just to allow other CWDs, and then I put `MANIFEST.in` and `setup.py` in my fork.

**Comment by jimbofreedman** [comment]
@iandees I've removed the packaging stuff

**Comment by iandees** [comment]
Apologies, I merged in #5838, which added more searchable points usages. Can you resolve the conflicts and I'll merge this.

**Comment by jimbofreedman** [comment]
I think this is ok?

**Pull request merged**